### PR TITLE
Links, tweets, handles - Twitter Sign Up flow

### PR DESCRIPTION
**Pull request by AvailableName1**
Before we had:
tweetTwitterBefore
tweetBefore

Now we will have:
tweetTwitterAfterSubmit
tweetTwitterAfter
tweetAfter

However I see:
_fckinErrorFromLocale
fckinErrorFromLocale

Which is because I'm trying from locale machine as I understand. **SO need to check from live site!!!**

### CODE REVIEW

**Comment by AvailableName1** [comment]
ALSO when signing message before:
image

Now:
image

File: src/components/twitter/TwitterFlow.tsx [hunk 1]
```diff
@@ -100,28 +120,51 @@ export default function TwitterFlow({ onDismiss }: { onDismiss: () => void }) {
...
+  // check if we need to add a custom emoji to tweet
+  const protocolEmojiFunc = (): string => {
+    if (!activeProtocol || !activeProtocol.emoji) return "👀";
+    if (activeProtocol.id === "AMPLIFI") return "";
+    return activeProtocol.emoji;
+  };
+  const protocolEmoji = protocolEmojiFunc();
+
+  // check if protocol has social or if it's APMPLIFI
+  const socialTagsFunc = (): string => {
```
Comment (jonomnom) [inline src/components/twitter/TwitterFlow.tsx:133]: @AvailableName1 clean code criticism: Don't name functions with func because its redundant.

Comment (jonomnom) [reply src/components/twitter/TwitterFlow.tsx:133]: Functions that are to do a function, should have a verb. Like `getEmoji` would work.

Comment (AvailableName1) [reply src/components/twitter/TwitterFlow.tsx:133]: > Functions that are to do a function, should have a verb. Like `getEmoji` would work.

ok, got it. I'll fix.

**Pull request merged**